src/Compiler/Microsoft.CodeAnalysis.Razor/src/SymbolVisibility.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.CodeAnalysis.Razor;
+
+internal enum SymbolVisibility
+{
+    Public = 0,
+    Internal = 1,
+    Private = 2,
+    Friend = Internal,
```

Comment:
We don't deal with VB, this alias isn't required.

Reply:
⏱️ Removing